src/test/java/com/icthh/xm/ms/entity/service/ProfileServiceUnitTest.java

```diff
@@ -89,4 +90,14 @@ public void getProfileByEntityId() {
 
         verify(profileRepository).findOneByXmentityId(entity.getId());
     }
+
+    @Test
+    public void deleteProfile() {
```

Comment:
need test that check db constrains did not break this operation
and test that check profile real was deleted